docs/source/pillars/supporting.md

```diff
@@ -0,0 +1,217 @@
+  - This may be due to downtime from service providers, a breach, or loss of power. Your plan should detail your process for managing loss of service for deployed TREs, and evaluation of impact of such loss
+  - Recommended
+* - You regularly test the aspects of your business continuity plan concerning TREs, and have a process in place to iterate the plan if required
+  -
+  - Recommended
+```
+
+## Project and programme management
+
+_What the organisation does to ensure effective management of programmes and projects._
+
+```{list-table}
+:header-rows: 1
+:name: tab-project-programme-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring TRE are assigned project management support
+  - This may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
+  - Recommended
+* - Project management support assigned to a project requiring a TRE should not have direct access to the TRE
+  - Doing so ensures a separation between those able to access sensitive data, and those overseeing access to sensitive data
+  - Optional
+```
+
+## Knowledge management
+
+_What the organisation does to acquire, enrich, share, store, publish and enhance the expertise across the whole organisation._
+
+```{list-table}
+:header-rows: 1
+:name: tab-knowledge-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All features of your TRE infrastructure requiring input or attention from any stakeholders must be well documented
+  - This includes ensuring all documentation is discoverable, clear, and able to be easily updated based on stakeholder feedback
+  - Mandatory
+* - You should have an education programme in place to upskill stakeholders in the use and management of your TRE infrastructure
+  - This may include learning modules, workshops and other resources on how to effectively access and use a TRE, FAQs pages, and accessible pathways for additional support
+  - Recommended
+* - You should periodically carry out a training needs analysis (TNA) for all stakeholders included within your TRE provision
+  - At least once every 12 months you should assess the training needs of your stakeholders, and ensure they have easy access to all required training materials
+  - Recommended
+```
+
+## Financial management
+
+_All activities aimed at the efficient and effective management of money (funds) in such a manner as to allow the organisation to accomplish its objectives._
+
+```{list-table}
+:header-rows: 1
+:name: tab-financial-management
+
+* - Statement
+  - Guidance
+  - Importance
+* - All projects requiring a TRE must know and accept the financial cost of doing so
+  - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
+  - Mandatory
```

Comment:
How about:

```suggestion
* - All projects using a TRE must have a plan for paying for any financial costs associated with their TRE use
  - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
  - Mandatory
```

Reply:
Ah I see the diff now - are we trying to make this as centred on TRE teams as poss (i.e. not hold researchers/users liable for requirements).

If so happy to commit, just need to ensure we define TRE operators somewhere

Reply:
(Added as a term to add to glossary, along with project manager & project management